projects/canopy/src/lib/forms/radio/radio-button.component.scss

```diff
@@ -1,5 +1,9 @@
 @import '../../../styles/mixins';
 
+lg-hint {
+  width: 100%;
```

Comment:
I wouldn't have thought you needed this? also could you target the class instead of the tag please?

Reply:
yeah, it was needed, otherwise, it stayed on the same row 😄

Reply:
Actually, upon better inspection, there was no need for me to add the styling, I thought it was needed because at first I had a different approach which didn't use the `lg-hint` so this styling part is out of the picture 😄